.github/workflows/ci.yml

```diff
@@ -61,7 +61,7 @@ jobs:
     needs:
       - build-info
     runs-on: ubuntu-20.04
-    timeout-minutes: 30
+    timeout-minutes: 60
```

Comment:
For testing CI this is fine. But let's revert it back once the compilation is hidden behind `-Pnative`.

Reply:
reverted done